mempool/reactor.go

```diff
@@ -271,10 +273,6 @@ func RegisterMessages(cdc *amino.Codec) {
 }
 
 func (memR *Reactor) decodeMsg(bz []byte) (msg Message, err error) {
-	maxMsgSize := calcMaxMsgSize(memR.config.MaxTxBytes)
-	if l := len(bz); l > maxMsgSize {
-		return msg, ErrTxTooLarge{maxMsgSize, l}
```

Comment:
Is this API breaking? Are there clients that may depend on `ErrTxTooLarge` being returned, in some form?

Reply:
```go
		memR.Logger.Error("Error decoding message", "src", src, "chId", chID, "msg", msg, "err", err, "bytes", msgBytes)
		memR.Switch.StopPeerForError(src, err)
```

no, we log this error and stop peer

Reply:
this is a private function, no external clients depend on it

Reply:
Yeah, I see that we have an additional check for this in the mempool.